src/popover/src/Popover.js

```diff
@@ -316,7 +315,8 @@ export default class Popover extends Component {
     } = this.props
     const { isShown: stateIsShown } = this.state
 
-    const shown = isShown || stateIsShown
+    // If `isShown` is a boolean, popover is controlled manually, not via mouse events
+    const shown = typeof isShown === 'boolean' ? isShown : stateIsShown
```

Comment:
I think this might break some of our consumers in the app that only use isShown for initial display (not using it as a controlled component)

Reply:
I don't understand, could you point to an example?

Reply:
I think we have some places where we set `setState({ isShown: true })` without also updating `setState({ isShown: false })` on cancel/confirm/close.

Reply:
I see, I'll check it out.

Reply:
I checked all cases where `<Popover>` is being used and I couldn't find any examples where `isShown` prop is being passed. Could you point me to those places you had in mind?

Reply:
I might have been thinking of `Dialog`

Reply:
Ah ok then.